The Game/Sprites/CactusDouble.py

```diff
@@ -0,0 +1,15 @@
+import pygame
+class CactusDouble(object):
+
+    def __init__(self, x, y):
+        self.x = x
+        self.y = y
+        self.hitbox = (0, 0, 0, 0)
+        self.imageName = "Sprites/GameImages/doubleCactus.png"
+
+    def drawCharacter(self, canvas):
+        canvas.blit(pygame.image.load(self.imageName), (self.x,self.y))
```

Comment:
Load the image once in constructor  and use it for blitting there after.

Reply:
changed whereever possible